internal/resources/helmfeature/helm_feature_resource.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package helmfeature
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/common"
+	commonscope "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/common/scope"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/helmfeature/scope"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/helmfeature/status"
+)
+
+const (
+	ResourceName = "tanzu-mission-control_helm_feature"
```

Comment:
nit: newline afterL18 for better readability

Reply:
Updated